File: mall-sellergoods-interface/src/main/java/com/marshal/sellergoods/service/BrandService.java
package com.marshal.sellergoods.service;


import com.marshal.pojo.TbBrand;
import com.marshal.util.ResponseData;
import java.util.List;

public interface BrandService {
    public List<TbBrand> findAll();

    public ResponseData query(TbBrand condition,int pageNum,int pageSize);

    public TbBrand queryById(Long id);

    public void save(TbBrand tbBrand);

    public void delete(Long[] ids);
}

File: mall-manager-web/src/main/webapp/js/service/brandService.js
// service层,与数据库交互
app.service("brandService",function($http){
    //BASE crud
    this.query=function(pageNum,pageSize,condition){
        return $http.post("../brand/query?pageNum="+pageNum+"&pageSize="+pageSize,condition);
    }

    this.save=function (brand) {
        return $http.post("../brand/save",brand);
    }

    this.remove=function (idList) {
        return $http.get("../brand/delete?idList="+idList);
    }

    this.queryById=function (id) {
        return $http.get("../brand/queryById?id="+id);
    }
});